shared/src/test/factories.ts

```diff
@@ -304,6 +304,7 @@ export const generateProfileData = (
     elevatorOwningBusiness: undefined,
     carnivalRideOwningBusiness: undefined,
     raffleBingoGames: undefined,
+    businessOpenMoreThanTwoYears: undefined,
```

Comment:
sand - shouldn't we add `isNonProfitFromFunding: undefined` to the relevant factory function?